### PR TITLE
Expect 200 for fee-estimation

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

#1096 

# Overview

- [x] I have added a `expectResponseCode HTTP.status200` for *STAKE_POOLS_ESTIMATE_FEE_01*
- [x] I tweaked the description of *STAKE_POOLS_ESTIMATE_FEE_04*

# Comments

### CODE REVIEW

File: lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs [hunk 1]
```diff
@@ -572,7 +573,7 @@ spec = do
             , expectErrorMessage $ errMsg404NoWallet (w ^. walletId)
             ]
 
-    describe "STAKE_POOLS_ESTIMATE_FEE_04 - wallet ids" $ do
+    describe "STAKE_POOLS_ESTIMATE_FEE_04 - invalid wallet ids" $ do
```
Comment (Anviking) [inline lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs]: So the wallet ids are either for a non-existing wallet, or totally wrong.

Maybe "false wallet ids" would be better (consistent with the `falseWalletIds` name) 🤷‍♂

**Comment by piotr-iohk** [comment]
swagger.yaml says it should be `200` -> https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getDelegationFee

I think that `200` suits better and is consistent with other endpoints that estimate fee (transactions and migrations).

**Comment by Anviking** [comment]
@piotr-iohk Thanks, good point, actually I was testing the response code on the completely wrong call.

The `delegationFee` is already returning 200. It is `joinStakePool` that returns 202.

I removed the 202 check on `joinStakePool` (it is tested in its own test), and added a 200 check for `delegationFee`.

File: lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs (hunk 1 above)
Comment (Anviking) [reply lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs]: Went with "false wallet ids".

**Review by KtorZ — APPROVED** [review]
bors r+

**Pull request merged**